storage/scanner.go

```diff
@@ -125,28 +125,35 @@ func (rs *rangeScanner) RemoveRange(rng *Range) {
 	rs.removed <- rng
 }
 
+// heartBeat returns a duration between iterations to allow us to pace
+// the scan.
+func (rs *rangeScanner) heartBeat(start time.Time) time.Duration {
```

Comment:
"Heartbeat" is one word; we don't capitalize the "B". `heartbeatInterval` would be a better name for this method.